addon/index.js

```diff
@@ -45,6 +44,8 @@ const WebExtensionChannels = require('./lib/webextension-channels');
 const ToolbarButton = require('./lib/toolbar-button');
 const ExperimentNotifications = require('./lib/experiment-notifications');
 
+const EXPERIMENT_UPDATE_INTERVAL = 24 * 60 * 60 * 1000; // 1 day
```

Comment:
Yeah, [looks like linting on CircleCI caught it](https://circleci.com/gh/mozilla/testpilot/675)